doc/Type/Mu.pod6

```diff
@@ -186,7 +186,7 @@ near future. Use C<.raku> instead.
     multi method raku(Mu:D:)
 
 For type objects, returns its name if C<.perl> has not been redefined from
```

Comment:
Er. Maybe it should say C<.raku>...

Reply:
It is my understanding that this description is accurate with respect to current rakudo's implementation. So I'm not sure what should be done here: keep an accurate description of the implementation or ... ?

Reply:
In general, we shouldn't take implementation details into account, but the intent of the class. Probably what would make sense is to describe the behavior without really referring to how the implementation is really made.

Reply:
This description had been meanwhile updated by 0143a7cb90ae0c8f75fca01028fb630a4931ee6f
Closing this for now.